src/bootstrapper.cc

```diff
@@ -51,6 +51,9 @@ void SetupNextTick(const FunctionCallbackInfo<Value>& args) {
 }
 
 void PromiseRejectCallback(PromiseRejectMessage message) {
+  static uint64_t unhandledRejections = 0;
+  static uint64_t rejectionsHandledAfter = 0;
```

Comment:
Are these values supposed to be global? Wouldn’t something per-Node.js-instance (aka per-`Environment`) make more sense?

(If they are supposed to be global: You probably want `std::atomic_uint64_t` instead.)

Reply:
Currently thinking that global is the right thing for now. This is really intended as a kind of spot check... e.g. let's do a really quick check to see the ratio of unhandled rejections to handled after rejections to see if we may have a problem. I don't think we need more detail than that. +1 on `std::atomic_uint64_t`